### ISSUE TITLE
Azure Monitor OpenTelemetry Exporter using depreciated semantic conventions

### ISSUE THREAD

**Issue by dasiths** [Azure/azure-sdk-for-python#34246]
- **Package Name**:  azure-monitor-opentelemetry-exporter
- **Package Version**:  latest
- **Operating System**: n/a/
- **Python Version**:  n/a

**Describe the bug**
The azure monitor opentelemetry exporter uses depreciated semantic conventions.

- https://github.com/Azure/azure-sdk-for-python/blob/54e0fa6e8d9ff214c023964a3d1b299819d12061/sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py#L351

Example:
Uses the `SpanAttributes.HTTP_METHOD` which is depreciated as per https://github.com/open-telemetry/opentelemetry-python/blob/d03a62245357eb40cac81d5e671fc2df83aef767/opentelemetry-semantic-conventions/src/opentelemetry/semconv/trace/__init__.py#L49

When using a span with `SpanKind.Client` you have to rely on the deprecated semantic conventions for Azure Monitor to pick it up.

```python
with tracer.start_as_current_span("OpenAI completion api request", kind=SpanKind.CLIENT) as span:
    span.set_attribute("http.method", "GET") 
    span.set_attribute("http.url", "https://my-deployed-service.openai.azure.com/")
    
   # or span.set_attribute("db.system", "my-deployed-service.openai.azure.com") 
```

**Comment by lzchen** [comment]
This is an issue for upstream OpenTelemetry. We are in the process of adding new semantic conventions to various http instrumentations. See the tracking issue [here](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1986). Once these are done, the exporters will be changed to allow for those new signals.

**Comment by dasiths** [comment]
> This is an issue for upstream OpenTelemetry. We are in the process of adding new semantic conventions to various http instrumentations. See the tracking issue [here](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1986). Once these are done, the exporters will be changed to allow for those new signals.

Thanks for the update. Is there a recommended workaround or migration guidance until this is completed? i.e. Do we use `db.system` or `az.namespace` attribute until then or keep using the deprecated http attributes?

**Comment by lzchen** [comment]
@dasiths 
Yes, please continue to use old semantic attributes until otherwise notified.

**Comment by lzchen** [comment]
Tracking issue: https://github.com/Azure/azure-sdk-for-python/issues/35933

**Comment by lzchen** [comment]
Tracking issue in OT upstream for migration plan for instrumentations: https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2453

**Comment by rads-1996** [comment]
Closing as this issue as the new semantic conventions have been incorporated in the SDK.
